CHANGELOG.md

```diff
@@ -15,8 +15,12 @@ and includes an additional section for migration notes.
 
 ## [Unreleased]
 
+## [9.0.0] 2023-10-05
+
 ### Migration Notes
 
+- Update terraform to the latest 1.5 version
+
 ### Added
 
```

Comment:
We should remove the unused headings from the release 9 heading 
### Added
### Changed
### Deprecated
### Removed
### Fixed

Since we don't have any items for those this time around.

We should add all the headings under unreleased so we have a spot to add items for the next release.

Reply:
Fixed.